content/pytorch/concepts/tensors/terms/zeros/zeros.md

```diff
@@ -0,0 +1,71 @@
+---
+Title: '.zeros()'
+Description: 'Initializes a new tensor filled with zeros, with a specified shape.'
+Subjects:
+  - 'AI'
+  - 'Data Science'
+  - 'Machine Learning'
+Tags:
+  - 'AI'
+  - 'Arrays'
+  - 'Data Structures'
+  - 'Deep Learning'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'py-torch-for-classification'
+  - 'paths/data-science'
```

Comment:
``` suggestion
CatalogContent:
  - 'intro-to-py-torch-and-neural-networks'
  - 'py-torch-for-classification'
  - 'paths/data-science'
```
Only use the course/path that is already present in the [catalog-content. md file ](documentation/catalog-content.md)and else if you do not have the paths that you want to include in the PR present in the list then add one.

Reply:
sorry i don't see the file, is there another link i can see what courses there are?

Reply:
oh i found it sorry nvm